Tutorials/Porting-Packages-V8/index.md

```diff
@@ -0,0 +1,178 @@
+# Porting Umbraco V7 packages to work with V8
+This is not a definitive guide for all scenarios and more of a working document and will evolve over time, especially whilst V8.0.0 is development and things in this guide may change.
+
+So you have a useful Umbraco package built for V7 and you would like to see it, work for V8?
+Well then this guide, should give you a few pointers to get started with the process and is mostly geared towards solutions that use C#
+
+## Upgrade .NET Framework
+Upgrade your Visual Studio C# project to use and compile against the .NET Framework 4.7.2. This is because the minimum to run version 8 of Umbraco requires this specific version of the .NET Framework
+
+## Add Nuget.config file to your solution
+As Umbraco V8 is still in development and not been published to the official Nuget.org feed, then to allow you to easily add references to the Umbraco DLLs in your project.
+
+Add the following `nuget.config` file to the root of your project, for Visual Studio to discover it
+
+  <?xml version="1.0" encoding="utf-8"?>
+  <configuration>
+    <!--
+      this is Umbraco's NuGet configuration,
+      content of this file is merged with the system-wide configuration,
+      at %APPDATA%\NuGet\NuGet.config
+    -->
+    <packageSources>
+      <add key="UmbracoCoreMyGet" value="https://www.myget.org/F/umbracocore/api/v3/index.json" />
+      <add key="ExamineAppVeyor" value="https://ci.appveyor.com/nuget/examine-f73l6qv0oqfh/" />
+    </packageSources>
+  </configuration>
+
+
+## Add references via Nuget
+You will need to add `UmbracoCms.Core` via Nuget from the `UmbracoCoreMyGet` feed with the pre-release flag enabled.
+
```

Comment:
I guess package devs would know the command for this, but maybe we should add it here just to be nice? 🤔
`Install-Package UmbracoCms.Core -pre -source https://www.myget.org/F/umbracocore/api/v3/index.json`

Reply:
Well I still use the UI in Visual Studio to add/manage my Nuget package references. But yes a valid point will add the command :)